Dockerfile

```diff
@@ -15,4 +15,8 @@ RUN systemctl set-default multi-user.target
 
 COPY setup /sbin/
 
-CMD ["/sbin/init"]
+STOPSIGNAL SIGRTMIN+3
+VOLUME ["/sys/fs/cgroup"]
```

Comment:
VOLUME was just more for self-documentation I guess, as a reminder that you need to mount it.